channels/management/commands/runserver.py

```diff
@@ -26,11 +26,14 @@ def add_arguments(self, parser):
             help='Run the old WSGI-based runserver rather than the ASGI-based one')
         parser.add_argument('--http_timeout', action='store', dest='http_timeout', type=int, default=60,
             help='Specify the daphane http_timeout interval in seconds (default: 60)')
+        parser.add_argument('--ws_open_handshake_timeout', action='store', dest='ws_open_handshake_timeout', type=int, default=5,
```

Comment:
Like I requested in https://github.com/django/daphne/pull/121, let's change this too.